### PR TITLE
release v0.2.0

### PR DESCRIPTION
**Pull request by ISulimanov**

### CODE REVIEW

File: object/check.go [hunk 1]
```diff
@@ -256,6 +268,10 @@ func CheckLdapUserPassword(user *User, password string, lang string) (string, er
 	userDisabled := false
 
 	for _, ldapServer := range ldaps {
+		if ldapId != "" && ldapServer.Id != ldapId {
+			continue
+		}
```
Comment (dplynskiy) [inline object/check.go:273]: if we have ldapId -> we can query only 1 ldapServer by id from db.

File: object/check.go [hunk 2]
```diff
@@ -371,62 +387,62 @@ func (err *CheckUserPasswordError) RealError() error {
...
+	isPasswordWithLdapEnabled = options.IsPasswordWithLdapEnabled
+	
 	user, err := GetUserByFields(organization, username)
 	if err != nil {
 		return nil, err
 	}
 
 	if user == nil || user.IsDeleted {
-		return nil, fmt.Errorf(i18n.Translate(lang, "general:The user: %s doesn't exist"), util.GetId(organization, username))
+		return nil, fmt.Errorf(i18n.Translate(options.Lang, "general:The user: %s doesn't exist"), util.GetId(organization, username))
```
Comment (dplynskiy) [inline object/check.go:405]: I think we should transfer Lang in context, because it's request scoped data.

File: ldap/server.go [hunk 3]
```diff
@@ -61,7 +61,10 @@ func handleBind(w ldap.ResponseWriter, m *ldap.Message) {
 		}
 
 		bindPassword := string(r.AuthenticationSimple())
-		bindUser, checkPassErr := object.CheckUserPassword(ctx, bindOrg, bindUsername, bindPassword, "en")
+		options := object.CheckUserPasswordOptions{
+			Lang: "en",
+		}
```
Comment (dplynskiy) [inline ldap/server.go:66]: why we use "en" instead c.GetAcceptLanguage() ?

**Review by dplynskiy — CHANGES_REQUESTED** [review]

File: object/check.go (hunk 1 above)
Comment (ISulimanov) [reply object/check.go]: done

File: object/check.go (hunk 2 above)
Comment (ISulimanov) [reply object/check.go:410]: let fix in another issue

**Review by dplynskiy — APPROVED** [review]

**Pull request merged**